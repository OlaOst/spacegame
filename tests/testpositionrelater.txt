parent.position = [1.0, 0.0]
parent.isRelation = true

child.position = [0.0, 0.0]
child.relationName = parent
child.relativePosition = [0.0, 1.0]