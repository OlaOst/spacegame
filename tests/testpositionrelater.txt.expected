parent.position = [1, 0]
parent.isRelation = true

child.position = [1, 1]
child.relationName = parent
child.relativePosition = [0, 1]
